tests/evergreen/cli/test_main.py

```diff
@@ -32,8 +33,8 @@
 TEST_AUTH_USERNAME = "test.user"
 
 
-@pytest.fixture(params=output_formats)
-def output_fmt(request):
+@pytest.fixture(name="output_fmt", params=output_formats)
+def output_fmts(request):
```

Comment:
Give it a name in order to avoid warning about parameter below shadowing global: https://pylint.readthedocs.io/en/stable/user_guide/messages/warning/redefined-outer-name.html